coretex/cli/modules/node.py

```diff
@@ -132,15 +134,17 @@ def start(dockerImage: str, config: Dict[str, Any]) -> None:
         raise NodeException("Failed to start Coretex Node.")
 
 
-def stop() -> None:
+def stop(echoOutput: bool = True) -> None:
```

Comment:
Should there be a new function which removes network and container? It isn't very clear why is stop being used if a container is not running but it exists.

Reply:
Now when I think a bit about it you are right it is confusing.